indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1724,11 +1726,11 @@ def __init__(self, conf, badgeTemplate, marginTop, marginBottom, marginLeft, mar
         self.__marginRight = marginRight
         self.__marginColumns = marginColumns
         self.__marginRows = marginRows
-
         if registrantList == 'all':
-            self.__registrantList = self.__conf.getRegistrantsList(sort = True)
+            self.__registrantList = Registration.find(~Registration.is_deleted).all()
```

Comment:
Ah, maybe just check if it belongs to the event for now.

    Registration.event_id == self.event.id